src/address.rs

```diff
@@ -1,116 +1,90 @@
 use crate::error::Error;
 use crate::SubnetID;
+use fil_actors_runtime::cbor;
+use fvm_ipld_encoding::RawBytes;
 use fvm_shared::address::Address;
 use fvm_shared::ActorID;
 use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
-use std::fmt;
-use std::fmt::Formatter;
 use std::str::FromStr;
 
-/// Maximum length for an address payload determined by
-/// the maximum size of the IPC address.
-const MAX_ADDRESS_LEN: usize = 54;
-
 // The default actor id namespace for IPC addresses
 lazy_static! {
     pub static ref DEFAULT_IPC_ACTOR_NAMESPACE_ID: ActorID = 1000u64;
 }
 
 #[derive(Clone, PartialEq, Eq, Debug, Hash, Serialize, Deserialize)]
 pub struct IPCAddress {
-    inner: Address,
-}
-
-impl fmt::Display for IPCAddress {
-    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        self.inner.fmt(f)
-    }
+    subnet_id: SubnetID,
+    raw_address: Address,
```

Comment:
```suggestion
    local_address: Address,
```

Reply:
What does `local` mean?

Reply:
"Local" address in the specific subnet.

Reply:
"Raw" could suggest unstructured representation of the whole address.

Reply:
Actually, "Local" is good. I am responsible for this "raw" terminology (we are using filecoin addresses and there is no unstructure representation of an address, you either have the IPC wrapper or the raw address). In any case, I am hoping to improve the design and terminology for IPC addresses with the outcome of [this discussion](https://github.com/protocol/ConsensusLab/discussions/190).